packages/integration/lib/src/views/feedroom/feedroom_screen.dart

```diff
@@ -604,7 +604,7 @@ class _LMFeedRoomScreenState extends State<LMFeedRoomScreen> {
                           ),
                           LMFeedButton(
                             onTap: () {
-                              newPostBloc.add(state.event!);
+                              // newPostBloc.add(state.event!);
```

Comment:
Uncomment this to add refresh functionality

Reply:
updated!